### PR TITLE
Updates weights for RetinaNet

### PR DESCRIPTION
**Pull request by LukeWood**
Much better scores- 0.36 MaP.

This was achieved by NOT freezing batchnorm, and using a slower LR schedule with more epochs.

From https://github.com/keras-team/keras-cv/pull/1691

### CODE REVIEW

**Review by jbischof — APPROVED** [review]
Still not 100% on the `.weights` story. Otherwise, great thanks!

**Review by ianstenbit — APPROVED** [review]
